common/add_operation_error.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+parent: Common
+---
+
+# Add Operation Error
+
+Understanding the `AddOperationError` exception.
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .warning }
+> _An **Add** operation targeting an existing profile could end up effectively performing an **Alter** operation on this profile. Pyracf will always raise an `AddOperationError` to bring attention to these failures before attempting the **Add** operation._
```

Comment:
```markdown
_An **Add** operation targeting an existing profile could end up effectively performing an **Alter** operation on an existing profile. Pyracf will always raise an `AddOperationError` and **refuse** to perform the operation to bring attention to this condition._
```

Reply:
Sure, agreed that this is better. Changed.

Reply:
change made

Reply:
Change `pyRacf` to `pyRACF`.

Reply:
Wow, sorry I missed that. fixed.